servers/curated-corpus-api/src/public/resolvers/queries/Section.ts

```diff
@@ -13,8 +13,10 @@ import { Section } from '../../../database/types';
 export async function getSections(
   parent,
   args,
-  { db }
+  context
 ): Promise<Section[]> {
   const { filters } = args;
-  return await dbGetSectionsWithSectionItems(db, filters.scheduledSurfaceGuid);
+  const sections = await dbGetSectionsWithSectionItems(context.db, filters.scheduledSurfaceGuid);
+  // only return sections that are not disabled
+  return sections.filter(section => section.disabled === false);
```

Comment:
curious why you chose to use the existing query and filter here instead of making a new query.

i think i'd lean towards making a query specifically for the public graph. the `active` filter will likely be configurable for admin usage as we build this feature out (editors will want to see in-progress sections). and we already know `disabled` will differ between public and admin graphs.

also, as the public query will be hit in (relatively) high volume by merino, i think a more optimized flow would be beneficial.

all that said - it may be worth making a ticket to add an index to the `Sections` table. (i don't think we need an index immediately, but if sections get traction it could be beneficial.) we can chat about this if you'd like. 😄